styleguide.config.js

```diff
@@ -1,130 +1,191 @@
 const path = require('path')
-const pkg = require('./package.json');
-const homePageComponents = ['src/lib/containers/home_page/**/[A-Z]*.tsx', 'src/lib/containers/widgets/themes-plot/**/ThemesPlot.tsx', 'src/lib/containers/**/UserCardListRotate.tsx', 'src/lib/containers/**/RssFeedCards.tsx', 'src/lib/containers/**/TableFeedCards.tsx', 'src/lib/containers/**/UpsetPlot.tsx', 'src/lib/containers/**/MarkdownSynapse.tsx']
-const ignoreComponents = ['**/*.Mobile.tsx','**/*.Desktop.tsx']
+const {
+  createWebpackDevConfig,
+  createWebpackProdConfig,
+} = require('@craco/craco')
+const { NormalModuleReplacementPlugin } = require('webpack')
+
+const pkg = require('./package.json')
+const cracoConfig = require('./craco.config.js')
+const webpackConfig =
+  process.env.NODE_ENV === 'production'
+    ? createWebpackProdConfig(cracoConfig)
+    : createWebpackDevConfig(cracoConfig)
+
+const homePageComponents = [
+  'src/lib/containers/home_page/**/[A-Z]*.tsx',
+  'src/lib/containers/widgets/themes-plot/**/ThemesPlot.tsx',
+  'src/lib/containers/**/UserCardListRotate.tsx',
+  'src/lib/containers/**/RssFeedCards.tsx',
+  'src/lib/containers/**/TableFeedCards.tsx',
+  'src/lib/containers/**/UpsetPlot.tsx',
+  'src/lib/containers/**/MarkdownSynapse.tsx',
+]
+const ignoreComponents = ['**/*.Mobile.tsx', '**/*.Desktop.tsx']
 
 module.exports = {
-    title: `Synapse React Components v${pkg.version}`,
-    tocMode: 'collapse',
-    pagePerSection: true,
-    skipComponentsWithoutExample: true,
-    styleguideComponents: {
-        Wrapper: path.resolve(__dirname, 'src/lib/containers/StyleGuidistComponentWrapper'),
+  // This should normally just be the webpackConfig object created above, but react-styleguidist does not yet support CRA v5
+  // Here's a workaround: https://github.com/styleguidist/react-styleguidist/issues/1910#issuecomment-1013763698
```

Comment:
Have to modify the webpack config for a CRA v5 workaround

Reply:
That's unfortunate.